FrontEnd/src/App.js

```diff
@@ -23,7 +23,7 @@ function App() {
   return (
     <BrowserRouter>
       <div className="App">
-        <Header isAuthorized={true}></Header>
+        <Header isAuthorized={sessionStorage.getItem("isAuthenticated")}></Header>
```

Comment:
I would use redux or react context for this. This property could stay same after login. Could you check it?

Reply:
My teammate Ilya will work with this task. It's just an idea-proposal for him. Should i remove it?

Reply:
Thank you. Done.